cmdLine/src/main/scala/topgun/cmdline/JfrEventNotFoundException.java

```diff
@@ -0,0 +1,101 @@
+package topgun.cmdline;
+
+import java.io.PrintStream;
+import java.io.PrintWriter;
+
+public class JfrEventNotFoundException extends Exception{
+    public JfrEventNotFoundException() {
+        super();
+    }
+
+    public JfrEventNotFoundException(String s) {
+        super(s);
+    }
+
+    public JfrEventNotFoundException(String s, Throwable throwable) {
+        super(s, throwable);
+    }
+
+    public JfrEventNotFoundException(Throwable throwable) {
```

Comment:
you dont need all of these constructors - removing this one and the empty one forces the user to write a message (which is a good thing)